qiita_pet/handlers/study_handlers/prep_template.py

```diff
@@ -77,3 +79,42 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown prep template action: %s' % action)
         self.write(result)
+
+
+class PrepFilesHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        study_id = self.get_argument('study_id')
+        prep_file = self.get_argument('prep_file')
+        # atype = self.get_argument('type')
+
+        # TODO: Get file types for the artifact type
+        # FILE TYPE IN POSTION 0 MUST BE DEFAULT FOR SELECTED
+        file_types = ['raw_fwd', 'raw_rev']
+
+        selected = []
+        not_selected = []
+        _, base = get_mountpoint("uploads")[0]
+        uploaded = get_files_from_uploads_folders(study_id)
+        prep = pd.read_table(join(base, study_id, prep_file), sep='\t')
+        if 'run_prefix' in prep.columns:
```

Comment:
This comment is still unresolved. The name of the variables here makes the illusion that this code is only used one we have per_sample info which is not the case. Per sample files, is just a special case when we have the column run_prefix. That will make code maintainability harder.

Use a name such `per_prefix`.